app/service/update-appeal-service.ts

```diff
@@ -374,34 +344,25 @@ export default class UpdateAppealService {
     }
 
     caseData.timeExtensions = [];
-    const previousAskForMoreTimes = appeal.previousAskForMoreTime;
 
-    if (previousAskForMoreTimes) {
-      previousAskForMoreTimes.forEach(askForMoreTime => {
-        this.addCcdTimeExtension(askForMoreTime, appeal, caseData);
-      });
-    }
     const askForMoreTime = appeal.askForMoreTime;
-    if (askForMoreTime && askForMoreTime.reason) {
+    if (askForMoreTime && askForMoreTime.reason && askForMoreTime.status !== 'inProgress') {
```

Comment:
This can now be askForMoreTime && askForMoreTime.status === 'submitted' thought it was at some point to be honest